app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -15,10 +15,14 @@ const getMojoTweets = gql`
     tweets {
       createdAt
       text
-      favoriteCount
-      retweetCount
-      media
-      link
+      interactions {
+        favoriteCount
+        retweetCount
+      }
+      media {
+        image
+        link
+      }
```

Comment:
This seems like an unnecessary change when you're going to later switch this to multiple images.

Reply:
Even though this is part of a commit to refactor into sub-sections?

Reply:
Hrm. Yeah, that makes sense :+1: